examples/protocol-kit/index.ts

```diff
@@ -0,0 +1,184 @@
+import { ethers } from 'ethers'
+import EthersAdapter from '@safe-global/safe-ethers-lib'
+import SafeServiceClient from '@safe-global/safe-service-client'
+import Safe, { SafeFactory, SafeAccountConfig } from '@safe-global/safe-core-sdk'
+import { SafeTransaction, SafeTransactionDataPartial } from '@safe-global/safe-core-sdk-types'
+
+// Run this file:
+// source examples/.env
+// npx ts-node examples/protocol-kit/index.ts
+
+// https://chainlist.org/?search=goerli&testnets=true
+const RPC_URL='https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161'
+const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
+
+// Initialize signers
+
+// Create a wallet instance from the sender's private key
+const owner1Signer = new ethers.Wallet(process.env.OWNER_1_PRIVATE_KEY!, provider)
+const owner2Signer = new ethers.Wallet(process.env.OWNER_2_PRIVATE_KEY!, provider)
+const owner3Signer = new ethers.Wallet(process.env.OWNER_3_PRIVATE_KEY!, provider)
+
+const ethAdapterOwner1 = new EthersAdapter({
+  ethers,
+  signerOrProvider: owner1Signer
+})
+
+const txServiceUrl = 'https://safe-transaction-goerli.safe.global'
+const safeService = new SafeServiceClient({ txServiceUrl, ethAdapter: ethAdapterOwner1 })
+let safeFactory: SafeFactory;
+let safeSdkOwner1: Safe;
+let safeAddress: string;
+
+// If you have an existing Safe, you can use it instead of deploying a new one
+const EXISTING_SAFE_ADDRESS ='0xF188d41FD181f94960C5451D7ff6FdbcDf201a71';
```

Comment:
```suggestion
let safeFactory: SafeFactory
let safeSdkOwner1: Safe
let safeAddress: string

// If you have an existing Safe, you can use it instead of deploying a new one
const EXISTING_SAFE_ADDRESS ='0xF188d41FD181f94960C5451D7ff6FdbcDf201a71'
```